flasc/visualization.py

```diff
@@ -56,12 +64,14 @@ def plot_with_wrapping(
         its vectorized properties. This can help reduce the size of a .pdf or
         .eps file when this figure is saved, at the cost of a potential
         reduction in image quality.
+        rasterized (bool, optional): Rasterize the plot/line and thereby remove
+        its vectorized properties. Defaults to False.
+
```

Comment:
No second line indent; also there is an incorrectly listed argument `rasterize` in this list.